chapters/en/slides/module4/module4_02.md

```diff
@@ -12,88 +12,95 @@ In this slide deck we will see how we can visualize the changes between two comm
-so let's explore how we can access this view in JupyterLab, the terminal, and on GitHub.
 
 ---
 
-## 🙌 Differences between commits: JupyterLab
+## Comparing commits
 
+<br>
+<br>
+<br>
 <br>
 <br>
 <br>
 
 <center>
 
-<img src='/module4/diff-commits-jl.png' width="600px" alt="404 image"/>
+<img src='/module4/diff-intro-comparison.png' width="850px" alt="404 image"/>
 
 </center>
 
-Notes:
-To see the difference between commits in JupyterLab,
-you first need to open the **History** tab.
-Select the commit you want to compare
-and click in the file icon with the `+` and `-` symbols on it as you can see in this slide.
-In case you don't see these icons,
-you need to click on the small down arrow that in the upper right corner of the commit.
+Notes: 
+In this slide you can see the comparison between two commits for a single file (`README.md`). 
+The version of the file displayed on the left side is the older (commit `d59b6bb`)
+and the version on the right is more recent (commit `aa2a019`)
+The green and red highlights indicate the lines that have changed between the commits:
+In the example, the phrase *"Hello world!* 
+has been introduced in line number four in the commit `aa2a019`.
 
-By default you are going to compare the information with the last commit.
+This type of comparison is called the "diff" (difference) between commits.
+and it can be helpful to understand what has changed.
 
----
+Let's explore how we can access this view in JupyterLab, the terminal, and GitHub.
 
+---
 ## 🙌 Differences between commits: JupyterLab
 
 <br>
-<br>
-<br>
+
 
 <center>
 
-<img src='/module4/diff-commits-distant-jl.png' width="600px" alt="404 image"/>
+<img src='/module4/diff-commits-example-jl.png' width="800px" alt="404 image"/>
 
 </center>
 
 Notes:
-If you want to compare against another commit than the most recent one, you can click the symbol of the file with the left pointing arrow in the newest commit and then the icon of the right pointing arrow in the oldest commit you want to compare against.
-A new tab will be open in the bottom of the screen. There you can select the file you would like to compare.
+Open the **History** tab in JupyterLab to see the differences between commits.
+
+1 - Once there, find the older commit you want to compare first
+and then click the file icon symbol with the left arrow.
+Repeat the process for the most recent commit that you want to compare,
+but this time you should click the file icon symbol with the right arrow.
+After doing so,
+a new section called **Compare `d59b6bb` and `aa2a019`**   
+will appear at the bottom of the **History** tab. 
+
+2 - In that section   
+you should choose the file for which you want to see the differences.  
+In our example,    
+we have only changed the `README.md` file.  
+The next step is to click the file icon with the `+` and `-` symbols   
+next to the name of the file.
+The differences between both commits will be displayed as a new file.  
```

Comment:
```suggestion
2 - In that section   
you should choose the file for which you want to see the differences.  
In our example,
we have only changed the `README.md` file.
The next step is to click the file icon with the `+` and `-` symbols
next to the name of the file.
The differences between both commits will be displayed as a new file.
```

A few paragraphs shows up with odd formatting in the transcript (happened on some other slides as well):

![image](https://user-images.githubusercontent.com/4560057/193627514-a68787a5-12f1-4393-b0c3-febf55c1f29c.png)

This is because there are trailing spaces that are interpreted as line breaks in Markdown and making sure there are no spaces at the end of the sentences will fix this.